cse-runner-app/src/main/java/com/farao_community/farao/cse/runner/app/configurations/MendrisioConfiguration.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.cse.runner.app.configurations;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+@ConfigurationProperties(prefix = "cse-cc-runner.mendrisio")
+@ConstructorBinding
+public class MendrisioConfiguration {
+    private final MendrisioCagnoLineProperties mendrisioCagnoLine;
+    private final String mendrisioPstId;
+    private final String mendrisioNodeId;
+
+    public MendrisioConfiguration(MendrisioCagnoLineProperties mendrisioCagnoLine, String mendrisioPstId, String mendrisioNodeId) {
+        this.mendrisioCagnoLine = mendrisioCagnoLine;
+        this.mendrisioPstId = mendrisioPstId;
+        this.mendrisioNodeId = mendrisioNodeId;
+    }
+
+    public MendrisioCagnoLineProperties getMendrisioCagnoLine() {
+        return mendrisioCagnoLine;
+    }
+
+    public String getMendrisioPstId() {
+        return mendrisioPstId;
+    }
+
+    public String getMendrisioNodeId() {
+        return mendrisioNodeId;
+    }
+
+    public static final class MendrisioCagnoLineProperties {
```

Comment:
I think it's not a good practice to declare an inner class as public. if it is used in a public scope better to take it out of this class. What do you think ?

Reply:
I feel like this solution is accepted, like [here](https://www.amitph.com/spring-boot-nested-configuration-properties/#ConfigurationProperties_using_Nested_Inner_Classes). The separate class is feasible as well but I will need to create two additional files/classes for MendrisioCagno and PiSa links, I feel like it makes things a bit heavy in terms of file number. But if you still think it is better to separate I will do :)